ddtrace/tracer.py

```diff
@@ -95,7 +95,7 @@ def context_provider(self):
         """Returns the current Tracer Context Provider"""
         return self._context_provider
 
-    def configure(self, enabled=None, hostname=None, port=None, dogstatsd_host=None,
+    def configure(self, enabled=None, hostname=None, port=None, path=None, dogstatsd_host=None,
```

Comment:
same here re: `path`.

Probably not a problem, just curious thoughts.

Reply:
Agreed